packages/keychain/src/components/connect/create/useCreateController.ts

```diff
@@ -143,13 +142,21 @@ export function useCreateController({
           }
 
           if (controllerNode && publicKey) {
-            await initController(
+            const controller = await createController(
+              origin!,
+              chainId!,
+              rpcUrl!,
               username,
               controllerNode.constructorCalldata[0],
               controllerNode.address,
               credentialId,
               publicKey,
             );
+
+            await controller.login(NOW + DEFAULT_SESSION_DURATION);
```

Comment:
If I understand right, this creates a wildcard session, but should this be done for every login? Even if policies are defined?